lib/ansible/modules/system/ufw.py

```diff
@@ -336,14 +353,19 @@ def ufw_version():
             if (ufw_major == 0 and ufw_minor >= 35) or ufw_major > 0:
                 cmd.append([params['comment'], "comment '%s'" % params['comment']])
 
-            execute(cmd)
+            (_, rules_dry, _) = execute(cmd)
+            if module.check_mode:
+                if rules_dry.find("Rules updated") != -1:
+                    changed = True
 
     # Get the new state
-    (_, post_state, _) = module.run_command(ufw_bin + ' status verbose')
-    (_, post_rules, _) = module.run_command("grep '^### tuple' /lib/ufw/user.rules /lib/ufw/user6.rules /etc/ufw/user.rules /etc/ufw/user6.rules")
-    changed = (pre_state != post_state) or (pre_rules != post_rules)
-
-    return module.exit_json(changed=changed, commands=cmds, msg=post_state.rstrip())
+    if module.check_mode:
+        return module.exit_json(changed=changed, commands=cmds)
+    else:
+        (_, post_state, _) = module.run_command(ufw_bin + ' status verbose')
```

Comment:
While you're at it, could you change `ufw_bin + ' status verbose'` to `[ufw_bin, 'status', 'verbose']`?